app/views/coronavirus_form/medical_equipment.html.erb

```diff
@@ -17,18 +17,13 @@
   heading: t('coronavirus_form.medical_equipment.title'),
   is_page_heading: true,
   name: "medical_equipment",
-  items: [
+  items: t('coronavirus_form.medical_equipment.options').map do |_, item|
     {
-      value: t('coronavirus_form.medical_equipment.options.option_yes.label'),
-      text: t('coronavirus_form.medical_equipment.options.option_yes.label'),
-      checked: session[:medical_equipment] == t('coronavirus_form.medical_equipment.options.option_yes.label'),
-    },
-    {
-      value: t('coronavirus_form.medical_equipment.options.option_no.label'),
-      text: t('coronavirus_form.medical_equipment.options.option_no.label'),
-      checked: session[:medical_equipment] == t('coronavirus_form.medical_equipment.options.option_no.label'),
-    },
-  ]
+      value: item[:label],
+      text: item[:label],
+      checked: session[:hotel_rooms] == item[:label],
```

Comment:
```suggestion
      checked: session[:medical_equipment] == item[:label],
```

:question:

Reply:
Oops!  Copy and pasted this from another controller.

Reply:
This is why we have code review especially at this time of day. :laughing:

Reply:
Couldn't agree more!